Content.Shared/Lock/LockSystem.cs

```diff
@@ -16,10 +16,7 @@
 using JetBrains.Annotations;
 using Robust.Shared.Audio.Systems;
 using Robust.Shared.Utility;
-using Content.Shared._NF.Trade.Components;
 using Content.Shared.Emag.Components;
-using System.Text;
-using Content.Shared.Storage;
```

Comment:
idk anymore

Reply:
"Oh hey this was using an import that doesn't exist anymore - oh, look, there are other unused imports."